Libraries/Core/InitializeCore.js

```diff
@@ -173,6 +173,8 @@ defineProperty(global, 'Headers', () => require('fetch').Headers);
 defineProperty(global, 'Request', () => require('fetch').Request);
 defineProperty(global, 'Response', () => require('fetch').Response);
 defineProperty(global, 'WebSocket', () => require('WebSocket'));
+defineProperty(global, 'Blob', () => require('Blob'));
+defineProperty(global, 'URL', () => require('URL'));
```

Comment:
These both have a potential impact on the initial bundle size. Are we defining them as globals for compatibility with the web spec?